resources/asciidoctor/lib/inline_callout/extension.rb

```diff
@@ -0,0 +1,69 @@
+require 'asciidoctor'
+
+include Asciidoctor
+
+##
+# Enables inline callouts which asciidoc supports but asciidoctor doesn't.
+# Filed as enhancement request at
+# https://github.com/asciidoctor/asciidoctor/issues/3037
+#
+# Usage
+#
+#   POST <1> /_search/scroll <2>
+#
+# NOTE: This isn't an asciidoctor extension so much as a hack. Just including
+# the file causes us to monkey patch its code into asciidoctor. By  default we
+# don't change asciidoctor, but if you set the `inline-callouts` attribute so
+# you need to *ask* for the change in behavior.
```

Comment:
I know I am kind of reviewing with the `-pedantic` flag on here but... I still have some issues.

1. The new sentence no longer really makes sense.

> By default we don't change asciidoctor, but if you set the `inline-callouts` attribute so you need to *ask* for the change in behavior.

If it helps read it out loud. It's not conveying a coherent meaning.

2. It kind of tells a lie.

> By default we don't change asciidoctor

As also discussed below, logging behavior is being altered too, and that behavior change is not guarded by any check against `(attr? 'inline-callouts')` either, it always does it.

Reply:
The logging behavior is only altered for the duration of the monkey patching which is right at startup. We restore the old verbose mode after completing the patch.

I'll see about rewriting the sentence.